examples/lang-err-integration-tests/constructors-return-value/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "constructors_return_value"
```

Comment:
I've added this "example" into your integration tests folder @HCastano, because already all the CI scripts and waterfall tests deal with this special case folder. Perhaps we could introduce an `integration-tests` folder and have feature specific integration tests in there.

Reply:
> Perhaps we could introduce an integration-tests folder and have feature specific integration tests in there.

Yep, I was thinking the same thing